ament_index_cpp/QUALITY_DECLARATION.md

```diff
@@ -121,7 +121,13 @@ A description of how coverage statistics are summarized from this page, can be f
 
 ### Performance [4.iv]
 
-`ament_index_cpp` does not conduct performance tests.
+An environment variable defines the prefix paths of such resource indices and the API has a time complexity of `O(n)` where `n` is the number of prefix paths.
+The time complexity to query information is either scaling lineraly with the number of resource types or with the number of resources per type (depending on which dimension is requested).
```

Comment:
@dirk-thomas nit: 
```suggestion
The time complexity to query information is either scaling linearly with the number of resource types or with the number of resources per type (depending on which dimension is requested).
```

Reply:
Thanks. Fixed in 567dc522ed7596a0f5cad69844e0fad0e8f02936.